cabal.project

```diff
@@ -100,6 +100,11 @@ write-ghc-environment-files: always
 package snap-server
   flags: +openssl
 
+package bitvec
+   flags: -simd
```

Comment:
This needs a comment, probably linking to https://github.com/input-output-hk/haskell.nix/issues/2047

Reply:
I've actually made some progress here: I've realised that we only need this because we're building `plutus-core` with `-fexternal-interpreter` to get around a weird profiling issue. I tried deleting that workaround, and it seems that the other weird bug has also disappeared at some point. Anyway, I'll make a PR with the simplification if I can verify it's all good.

Reply:
(PR to master, you should just merge this one)